<launch>
  <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen"
       clear_params="true">
    <rosparam command="load" file="$(find lesson6_package_pcl)/params/octomap_mapping.yaml" />
    <remap from="/cloud_in" to="/laser_cloud" />
    <remap from="/octomap_point_cloud_centers" to="/octomap_cloud" />
  </node>
</launch>
